easybuild/easyconfigs/m/mpmath/mpmath-1.1.0-GCCcore-9.3.0.eb

```
@@ -0,0 +1,46 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author: Adam Huffman
+# adam.huffman@crick.ac.uk
+# The Francis Crick Institute
+
+easyblock = 'PythonPackage'
+
+name = 'mpmath'
+version = '1.1.0'
```

@SimonPinches please add versionsuffix (also to the file name):
```suggestion
version = '1.1.0'
versionsuffix = '-Python-%(pyver)s'
```

---

@smoors I've fixed this in the latest commit.